src/connectors/infinite-hits/__tests__/connectInfiniteHits-test.ts

```diff
@@ -88,7 +87,7 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/infinite-hi
         showMore: expect.any(Function),
         results: expect.any(Object),
         isFirstPage: true,
-        isLastPage: false,
```

Comment:
without the mock of the search response it got this wrong, since nbPages should be the same as page (both 0)